[BUG] offline trancoding fail After 2.5

Hey Alex,

after the latest Update the offline transcoding fails everytime. Playback is Fine. 

https://gist.github.com/Fabian1601/b8ae614a42c24be7030ca1a8b8bb79de

% Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0
100  9965  100  9965    0     0  18044      0 --:--:-- --:--:-- --:--:-- 18052
[0;37m[2023-10-23 21:45:54] [0m[INFO] [0mffmpeg-patcher
[0;37m[2023-10-23 21:45:54] [0m[INFO] [0mDownloading motd... [0;31mError
[0;37m[2023-10-23 21:45:54] [0m[ERROR] [0;31mAn error occurred while downloading https://raw.githubusercontent.com/AlexPresso/VideoStation-FFMPEG-Patcher/fix_pixfmt/motd.txt. Rolling back changes...
[0;37m[2023-10-23 21:45:54] [0m[INFO] [0m====== Unpatch procedure ======
[0;37m[2023-10-23 21:45:54] [0m[INFO] [0mlibsynovte.so was not patched, keeping actual file.
[0;37m[2023-10-23 21:45:54] [0m[INFO] [0mRemove patch config.
[0;37m[2023-10-23 21:45:54] [0m[INFO] [0mRestarting CodecPack...
restart package [CodecPack] successfully
[0;37m[2023-10-23 21:46:01] [0m[INFO] [0mRestarting VideoStation...
restart package [VideoStation] successfully
[0;37m[2023-10-23 21:46:15] [0m[INFO] [0mCleaning orphan files...
[0;37m[2023-10-23 21:46:15] [0m[SUCCESS] [0;32mUnpatch complete
[0;37m[2023-10-23 21:46:15] [0m[ERROR] [0;31mAn error occurred while downloading https://raw.githubusercontent.com/AlexPresso/VideoStation-FFMPEG-Patcher/fix_pixfmt/motd.txt, every changes were rolled back.
[0;37m[2023-10-23 21:46:15] [0m[ERROR] [0;31mPlease check your internet connection / GithubStatus. If you think this is an error, please file an issue to the repository.

--- a/motd.txt
+++ b/motd.txt
@@ -1,4 +1,4 @@
-A new version of the wrapper was deployed (v2.5) fixing some issues with hardware transcoding.
-You can find the changelog here: https://github.com/AlexPresso/VideoStation-FFMPEG-Patcher/releases/tag/2.5
+A new version of the wrapper was deployed (v2.6) fixing an issue with offline transcoding.
+You can find the changelog here: https://github.com/AlexPresso/VideoStation-FFMPEG-Patcher/releases/tag/2.6
 
 As always, please let me know of any issue you could have.

--- a/wrappers/ffmpeg.sh
+++ b/wrappers/ffmpeg.sh
@@ -122,6 +122,7 @@ if [[ $errcode -eq 0 ]]; then
   endprocess
 fi
 
+errcode=0
 info "Trying with VideoStation's ffmpeg with default args..."
 /var/packages/VideoStation/target/bin/ffmpeg.orig "$@" <&0 2>> $stderrfile &
 child=$!
@@ -131,6 +132,7 @@ if [[ $errcode -eq 0 ]]; then
   endprocess
 fi
 
+errcode=0
 info "Trying with SC's ffmpeg and fixed args..."
 "/var/packages/${ffmpeg_version}/target/bin/ffmpeg" "${args[@]}" <&0 2>> $stderrfile &
 child=$!

--- a/wrappers/ffmpeg.sh
+++ b/wrappers/ffmpeg.sh
@@ -87,10 +87,6 @@ fix_args() {
         shift
         ;;
 
-      -pix_fmt)
-        shift
-        ;;
-
       *) args+=("$1") ;;
     esac
 

